src/server_manager/web_app/test_action.sh

```diff
@@ -14,4 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+[[ -e ${ROOT_DIR}/src/server_manager/install_scripts/do_install_script.ts ]] \
```

Comment:
This seems misplaced. Where do we build the tests?

Reply:
That's there so that if we run server_manager/test we don't build the install scripts twice.  It's needed if we directly run `yarn do web_app/test`.  `karma.conf.js` calls into webpack, which builds the tests.